lms/static/lms/js/require-config.js

```diff
@@ -210,6 +210,13 @@
             },
             "afontgarde": {
                 exports: "AFontGarde"
+            },
+            "edxicons": {
+                exports: "edxicons"
+            },
+            "draggabilly": {
```

Comment:
Add a comment here stating that:

Because Draggabilly is being used by video code, the namespaced version of require is not being recognized. Therefore the library is being added to the global namespace instead of being registered in require.